### PR TITLE
Tlmblgis 158

### PR DESCRIPTION
**Pull request by dovek**
Due: 10/7
@lsloan 
@dlhaines

### CODE REVIEW

**Comment by dlhaines** [comment]
Looks ok to my innocent eyes.

File: hacks_mbof/settings.py [hunk 1]
```diff
@@ -171,16 +171,24 @@
         'simple': {
             'format': '%(levelname)s %(name)s %(message)s'
         },
+        'ocellus_formatter': {
+            'format': '%(levelname)s %(asctime)s %(module)s %(message)s',
```
Comment (lsloan) [inline hacks_mbof/settings.py:175]: I prefer the timestamp to be first on the line, but if you think this works better for some reason, it's OK.

File: hacks_mbof/settings.py [hunk 2]
```diff
@@ -171,16 +171,24 @@
...
         },
+        'ocellus_handler': {
+            'class': 'logging.StreamHandler',
+            'formatter': 'ocellus_formatter',
+        },
     },
     'loggers': {
         '': {
-            'handlers': ['console'],
+            # 'handlers': ['console'],
```
Comment (lsloan) [inline hacks_mbof/settings.py]: Remove commented-out line.  Was `console` ever helpful?  Helpful enough to be immortalized in a comment?

File: mbofui/app/js/controllers.js [hunk 3]
```diff
@@ -17,24 +17,19 @@ ocellus.controller('mapController', ['$compile', '$scope', '$rootScope','$filter
...
-          $rootScope.currentViewUrl = '/api/events/' + $rootScope.currentView + '/';
-        }
-        else {
-          $rootScope.currentView = 'current';
-          $rootScope.currentViewUrl = '/api/events/' + $rootScope.currentView + '/';
-          getEvents('/api/events/' + $rootScope.currentView + '/');
-        }
-      });
+      // it fires after we have a user so that we can determine
+      // which belong to the current user - temporary
```
Comment (lsloan) [inline mbofui/app/js/controllers.js:21]: Is it really still temporary?

**Pull request merged**